packages/ui/react/src/components/Primitives/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,104 @@
+import type { MouseEventHandler } from 'react'
+
+import { ContainerPagination, PaginationContent } from './Pagination.styled'
+import { Button } from 'src/components/Buttons'
+
+type ConfigurationActionsProps = {
+  /** Disable all pagination actions */
+  disableAllActions?: boolean
+  /** Disable only previous/back pagination actions */
+  disablePreviousActions?: boolean
+  /** Disable only next/forward pagination actions */
+  disableNextActions?: boolean
+  /** Hide first page and last page buttons */
+  hideFirstAndLastButton?: boolean
+}
+
+type PaginationActionsProps = ConfigurationActionsProps & {
+  /** Handler for navigating to first page */
+  firstPaginateAction: MouseEventHandler<HTMLButtonElement>
+  /** Handler for navigating to previous page */
+  previousPaginateAction: MouseEventHandler<HTMLButtonElement>
+  /** Handler for navigating to next page */
+  nextPaginateAction: MouseEventHandler<HTMLButtonElement>
+  /** Handler for navigating to last page */
+  lastPaginateAction: MouseEventHandler<HTMLButtonElement>
+}
```

Comment:
Following the same naming/export pattern for other components, can you apply the changes below?

```suggestion
type PaginationProps = {
  /** Disable all pagination actions */
  disableAllActions?: boolean
  /** Disable only previous/back pagination actions */
  disablePreviousActions?: boolean
  /** Disable only next/forward pagination actions */
  disableNextActions?: boolean
  /** Hide first page and last page buttons */
  hideFirstAndLastButton?: boolean
  /** Handler for navigating to first page */
  firstPaginateAction: MouseEventHandler<HTMLButtonElement>
  /** Handler for navigating to previous page */
  previousPaginateAction: MouseEventHandler<HTMLButtonElement>
  /** Handler for navigating to next page */
  nextPaginateAction: MouseEventHandler<HTMLButtonElement>
  /** Handler for navigating to last page */
  lastPaginateAction: MouseEventHandler<HTMLButtonElement>
}
```